Automattic/woocommerce-payments
# Add a device type filter to the Transactions list page

Fixes #5756

#### Changes proposed in this Pull Request
Adding a Tap to Pay device source filter to the Transactions list page.

<img width="750" alt="Screenshot 2023-05-15 at 14 23 55" src="https://github.com/Automattic/woocommerce-payments/assets/8667118/9827757d-cf77-47d4-99a5-9a028d1c13f2">


#### Testing instructions
- One the server use 3371-gh-Automattic/woocommerce-payments-server PR
- Create orders paid with Tap to Pay.
- Navigate to Payment -> Transactions.
- Add a Device Type advanced filter.
- Verify the transactions in the list are only the Tap to Pay.
- Verify the transactions summary and pagination are working correctly.

-------------------


**Post merge**
